classic_cp/kidneyexchange/kidneyexchange.jl

```diff
@@ -0,0 +1,278 @@
+using SeaPearl
+
+"""
+Instances format
+4 0.33 #numberOfPairs density
+2 4    #pair 1 can receive a kidney from pairs 2 and 4
+1      #pair 2 can receive a kidney from pair 1
+4      #pair 3 can receive a kidney from pair 4
+3      #pair 4 can receive a kidney from pair 3
+"""
+
+struct InputData
+    numberOfPairs         :: Int
+    density               :: Float16
+    compatibilities       :: Vector{Union{Vector{Int}, Nothing}}
```

Comment:
what does a `Nothing` means ?

Reply:
There is no reason to use "nothing", I has just corrected it